t/maxminddb_test_helper.c

```diff
@@ -17,12 +17,10 @@
 #include <unistd.h>
 #endif
 
-#define NO_PROTO
-
 #ifndef strndup
 /* *INDENT-OFF* */
```

Comment:
We could remove this as well I think. There's one in maxminddb_test_helper.c too!

Reply:
Oh I'm commenting on that one. There's one in maxminddb-compat-util.h.

Reply:
I think these were here because it is copied code and we wanted to keep it identical to the code we copied. I don't have strong feelings on that, but that is why I left them.

Reply:
Oh ok. In that case it seems fine to leave them!